fixtures/stairville/xbrick-quad-16x8w-rgbw.json

```diff
@@ -0,0 +1,797 @@
+  "shortName": "xBrick RGBW",
+  "categories": ["Color Changer", "Strobe"],
+  "meta": {
+    "authors": ["Maximilian B."],
+    "createDate": "2023-02-11",
+    "lastModifyDate": "2023-02-11"
+  },
+  "comment": "4 Clusters of 4 RGBW LEDs (4x4)",
+  "links": {
+    "productPage": [
+      "https://www.thomann.de/stairville_xbrick_quad_16x8w_rgbw.htm"
+    ],
+    "manual": [
+      "https://images.static-thomann.de/pics/atg/atgdata/document/manual/c_385641_v4_en_online.pdf",
+      "https://images.thomann.de/pics/atg/atgdata/document/manual/c_385641_r4_en_online.pdf"
+    ],
+    "video": [
+      "https://video2.thomann.de/vidiot/02591c1c/video_i5704p10_yd59vqpa.mp4"
+    ]
+  },
+  "physical": {
+    "dimensions": [315, 88, 195],
+    "weight": 4.3,
+    "power": 125,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "lens": {
+      "degreesMinMax": [25, 25]
+    }
+  },
+  "matrix": {
+    "pixelKeys": [
+      [
+        ["1A", "1B", null, "2A", "2B", null, "3A", "3B", null, "4A", "4B"],
+        ["1C", "1D", null, "2C", "2D", null, "3C", "3D", null, "4C", "4D"]
+      ]
+    ],
+    "pixelGroups": {
+      "All": "all",
+      "1": { "name": ["^1[A-D]$"] },
+      "2": { "name": ["^2[A-D]$"] },
+      "3": { "name": ["^3[A-D]$"] },
+      "4": { "name": ["^4[A-D]$"] }
+    }
+  },
+  "templateChannels": {
+    "Red $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    }
+  },
+  "availableChannels": {
+    "Intensity red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Intensity green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Intensity blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "Intensity white": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Stroboscope effect": {
+      "capability": {
+        "type": "ShutterStrobe",
+        "shutterEffect": "Strobe",
+        "helpWanted": "What are min/max strobe speeds?",
+        "speedStart": "0%",
+        "speedEnd": "100%"
+      }
+    },
+    "Function select": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 4],
+          "type": "NoFunction",
+          "switchChannels": {
```

Comment:
This fixture has more switching channel craziness. Please could you check it out for me @FloEdelmann

Reply:
I'm seeing the error `Error: Channel 'NoFunction' is referenced more than once`. This is because I've used `NoFunction` to indicate that the channel is disabled. Is there a better way to do this?

Reply:
Unfortunately, without #222, there is no clean way to solve this yet. Either create separate `NoFunction` channels so that no channel is reused in one mode, or "extend" the ranges of the "real" channels, i.e. use e.g. "Intensity red" directly instead of switching between it and a `NoFunction` channel. That makes the switching channels implicit and less accurate, but probably has better software support anyway.

Reply:
Ok, for now I have just added additional channels.